themes/default/theme.lua

```diff
@@ -97,6 +97,20 @@ theme.titlebar_maximized_button_focus_inactive  = themes_path.."default/titlebar
 theme.titlebar_maximized_button_normal_active = themes_path.."default/titlebar/maximized_normal_active.png"
 theme.titlebar_maximized_button_focus_active  = themes_path.."default/titlebar/maximized_focus_active.png"
 
+-- Define custom titlebar tooltips 
+theme.titlebar_tooltip_messages = {
```

Comment:
If this is a copy of the default one, I would rather not re-define it in the theme. People fork/copy themes rather often. That means if we add new buttons later, they will be stuck with whatever was defined when they copied the theme.

Reply:
Default theme reverted. Described in documentation with usage example.